src/nIsForName.js

```diff
@@ -22,7 +22,9 @@
  * @return {string[]}
  */
 function nIsForName(names) {
-  // write code here
+  return names.filter(item => {
+    return item.split(' ').some(el => el[0] === 'N');
```

Comment:
no need to write return again, short version is enough

Reply:
Fixed!